recipes/psiclass/meta.yaml

```diff
@@ -8,17 +8,20 @@ source:
   sha256: c60c0f33f9a076b97ccdba68b9140a5fb6936a5c5e309d70b3d0cf558828f73d
 
 build:
-  number: 2
+  number: 3
+  run_exports:
+    - {{ pin_subpackage("myrecipe", max_pin="x.x") }}
 
 requirements:
   build:
     - make
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
   host:
-    - samtools 0.1.19.*
+    - samtools >= 1.10
```

Comment:
Can you try if adding `zlib` works? We could test by adding an samtools call to the tests, or maybe test the Galaxy tool against the generated artifacts ...

Reply:
Sure!

Reply:
Yes, it seem to work when using zlib